packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -400,6 +400,20 @@ In [API Access Management](/docs/concepts/api-access-management/) custom authori
 To include a granted scope array and convert it to a space-delimited string, use the following expression:<br>
 `String.replace(Arrays.toCsvString(access.scope),","," ")`
 
+## Expressions in group rules
+
+Group rule conditions only allow `String`, `Arrays`, and `user` expressions.
+
+For example, the following expressions are allowed in group rule conditions:
+
+* `user.hasBadge`
```

Comment:
I need to check other doc but do we need to specify `hasBadge` is one of the custom property defined in schema? Same thing for `favoriteColors` is array property in custom schema?

Reply:
I will let @brentschaus-okta make the call.

Reply:
I think it might be a good idea to use properties in this table and mention that these expression examples are based on the assumption that the user profile contains the properties mentioned in that table(?)

Reply:
I made an update.